### PR TITLE
add postman tests

### PR DESCRIPTION
**Pull request by davixcky**
- add basic postman check
- add postman runner
- add basic postman test

### CODE REVIEW

File: .signadot/testing/integration-test-job.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    # Run all postman tests
+    cd hotrod
+    
+    newman run postman/hotrod.postman_collection.json
+
+  routingContext:
+    sandbox: "@{sandbox}"
+  uploadArtifact:
+    - path: hotrod/playwright-report/index.html
```
Comment (foxish) [inline .signadot/testing/integration-test-job.yaml:22]: These paths should be postman

File: .signadot/testing/integration-test-job.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+spec:
```
Comment (foxish) [inline .signadot/testing/integration-test-job.yaml:1]: please rename this file to postman-test-job.yaml

File: postman/hotrod.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+							"mode": "raw",
+							"raw": "{\"sessionID\":1490,\"requestID\":3,\"pickupLocationID\":123,\"dropoffLocationID\":1123}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://frontend.hotrod-istio.svc:8080/dispatch",
```
Comment (foxish) [inline postman/hotrod.postman_collection.json:156]: It appears to be hardcoding this variable - can you make this var be derived from the @{namespace} similar to what you did with playwright?

**Review by foxish — COMMENTED** [review]
When I tried to run this as `signadot job submit --attach -f ./.signadot/testing/integration-test-job.yaml --set branch=main --set namespace=hotrod-istio --set sandbox=`, I got:

```
error: collection could not be loaded
  unable to read data from file "postman/hotrod.postman_collection.json"
  ENOENT: no such file or directory, open 'postman/hotrod.postman_collection.json'
```
Is there some missing change?

**Comment by davixcky** [comment]
> When I tried to run this as `signadot job submit --attach -f ./.signadot/testing/integration-test-job.yaml --set branch=main --set namespace=hotrod-istio --set sandbox=`, I got:
> 
> ```
> error: collection could not be loaded
>   unable to read data from file "postman/hotrod.postman_collection.json"
>   ENOENT: no such file or directory, open 'postman/hotrod.postman_collection.json'
> ```
> Is there some missing change?

You are using the wrong branch, you should use `branch=add-postman-tests`

**Review by foxish — APPROVED** [review]
LGTM, thanks

**Pull request merged**